backend/Lingva.DataAccessLayer/Context/EventMembersContext.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Lingva.DataAccessLayer.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Lingva.DataAccessLayer.Context
+{
+    public class ApplicationContext : DbContext
+    {
+        public DbSet<EventMembers> EventMembers { get; set; }
+
+        public ApplicationContext()
+            : base()
+        {
+            Database.EnsureCreated();
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer("Server=(localdb)\\mssqllocaldb;Database=Lingvaappdb;Trusted_Connection=True;");
```

Comment:
Never hard code a sql connection string in a repository.
Pass It through class constructor and get from app. settings